.github/workflows/ci_test.yml

```diff
@@ -0,0 +1,23 @@
+name: Python package
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, macOS-10.15, windows-2019]
+        python: [3.6, 3.7, 3.8, 3.9]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+
+      - name: Install Tox
+        run: pip install tox
+
+      - name: Run Tox
+        run: tox -e py
```

Comment:
Why tox and not just pytest?

Reply:
True in this case it could be just pytest. I adapted this from another project in which tox did make things simpler. Anyway the tox config in setup.cfg is good because in a local clone testing for all python versions is as simple as running `tox`.